tendrl/gluster_integration/objects/volume/atoms/rebalance_not_running/__init__.py

```diff
@@ -0,0 +1,71 @@
+import etcd
+import subprocess
+
+from tendrl.commons.event import Event
+from tendrl.commons.message import Message
+from tendrl.gluster_integration import objects
+from tendrl.gluster_integration.objects.volume import Volume
+
+
+class RebalanceNotRunning(objects.BaseAtom):
+    obj = Volume
+    def __init__(self, *args, **kwargs):
+        super(RebalanceNotRunning, self).__init__(*args, **kwargs)
+
+    def run(self):
+        Event(
+            Message(
+                priority="info",
+                publisher=NS.publisher_id,
+                payload={
+                    "message": "Checking if rebalance is not running"
+                },
+                request_id=self.parameters["request_id"],
+                flow_id=self.parameters["flow_id"],
+                cluster_id=NS.tendrl_context.integration_id,
+            )
+        )
+        try:
+            rebal_status = NS.etcd_orm.client.read(
+                'clusters/%s/Volumes/%s/rebal_status' % (
+                    NS.tendrl_context.integration_id,
+                    self.parameters['Volume.vol_id']
+                )
+            ).value
+            if rebal_status is not None:
+                if rebal_status == "not applicable" or\
```

Comment:
Do we have not_started state as well? I remember it was "not applicable" if not started at all and "completed" if earlier one is complete. If "not_started" as well there, can be added in conditions here.